types.go

```diff
@@ -74,7 +78,9 @@ type Field struct {
 	MapKeyType    string `json:"mapKeyType,omitempty" yaml:"mapKeyType,omitempty"`
 	MapValueType  string `json:"mapValueType,omitempty" yaml:"mapValueType,omitempty"`
 
-	StructFields map[string]Field `json:"structFields,omitempty" yaml:"structFields,omitempty"`
+	StructFields              map[string]Field              `json:"structFields,omitempty" yaml:"structFields,omitempty"`
+	StructFieldBindingTags    astTraversal.BindingTagMap    `json:"structFieldBindingTags" yaml:"structFieldBindingTags"`
```

Comment:
Can these be stored to json or etc? Therefore it would work with our caching method

Reply:
I think I might just need to add some tags to them but yeah it should be ok to be sorted by out caching mechanism.

Reply:
I have added the tags so it should work with our caching mechanism.